rules/windows/credential_access_seenabledelegationprivilege_assigned_to_user.toml

```diff
@@ -0,0 +1,64 @@
+[metadata]
+creation_date = "2022/01/27"
+maturity = "production"
+updated_date = "2022/01/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the assignment of the SeEnableDelegationPrivilege sensitive right to a user.  SeEnableDelegationPrivilege right enable computer and user accounts to be trusted for delegation. Attackers can abuse this right to compromise Active Directory accounts and elevate their privileges.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Sensitive Privilege SeEnableDelegationPrivilege assigned to a User"
+note = """## Config
+
+The 'Audit Authorization Policy Change' logging policy must be configured for (Success, Failure).
+Steps to implement the logging policy with Advanced Audit Configuration:
+
+```
+Computer Configuration >
+Windows Settings >
+Security Settings >
+Advanced Audit Policy Configuration >
+Audit Policies >
+Policy Change >
+Audit Authorization Policy Change (Success,Failure)
+```
+"""
+references = [
+    "https://www.harmj0y.net/blog/activedirectory/the-most-dangerous-user-right-you-probably-have-never-heard-of",
+    "https://github.com/SigmaHQ/sigma/blob/master/rules/windows/builtin/security/win_alert_active_directory_user_control.yml",
+    "https://github.com/atc-project/atomic-threat-coverage/blob/master/Atomic_Threat_Coverage/Logging_Policies/LP_0105_windows_audit_authorization_policy_change.md",
+]
+risk_score = 73
+rule_id = "f494c678-3c33-43aa-b169-bb3d5198c41d"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.code:4704 and winlog.event_data.PrivilegeList:"SeEnableDelegationPrivilege"
```

Comment:
I have been thinking that for these rules reliant on just the event code, that maybe we should add the `event.action` as well. While it is not needed for the logic to accurately detect the activity, it does add some detail to what may be opaque logic. Thoughts on adding it to this and other similar rules @w0rk3r?